### DISCUSSION TITLE
List users by multiple programming languages (API)

### DISCUSSION THREAD

**Discussion by siduck** [community/community#34735] [Apps, API and Webhooks]
Hi there, how do I list users by multiple programming languages?

for example this  `https://api.github.com/search/users?q=language:js` will list all users using `js` language

`https://api.github.com/search/users?q=language:js+language:css` will list all js users + css users

How do I make it list users that use both js and css?

**Comment by jfullstackdev** [comment]
just a quick thought, if there is no such endpoint to be consumed, what you can do is get JS and CSS, then look for matching people, and there you have it, you get the users using both CSS and JS

just my doubt is that, it will return more than thousand results so you need to be efficient in handling them

**Reply by siduck** [reply]
@jdevstatic  does this work? `https://api.github.com/search/users?q=language:lua+css` Im not sure how to verify it tho
